torch/_torch_docs.py

```diff
@@ -1901,6 +1901,12 @@ def merge_dicts(*dicts):
     y_i = \min(\max(x_i, \text{min\_value}), \text{max\_value})
 """ + r"""
 
+.. note::
+    If :attr:`min` is greater than :attr:`max` :func:`torch.clamp(..., min, max) <torch.clamp>`
+    sets all values equal to :attr:`max`.
```

Comment:
"all values" -> "all output values"?

Reply:
Maybe "sets all elements in `input` to `max`." Would be better to be more consistent with the general description that `torch.clamp()` 

> Clamp all elements in input into the range...